src/main/java/org/openrewrite/java/testing/jmockit/ExpectationsBlockRewriter.java

```diff
@@ -118,21 +119,25 @@ private void rewriteMethodBody(List<Statement> expectationStatements) {
             return;
         }
         J.MethodInvocation invocation = (J.MethodInvocation) expectationStatements.get(0);
-        if (!mockInvocationResults.getResults().isEmpty()) {
+        boolean hasExpectationsResults = !mockInvocationResults.getResults().isEmpty();
+        if (hasExpectationsResults) {
             // rewrite the statement to mockito if there are results
             rewriteExpectationResult(mockInvocationResults.getResults(), invocation);
         } else if (nextStatementCoordinates.isReplacement()) {
             // if there are no results and the Expectations block is not yet replaced, remove it
             removeExpectationsStatement();
         }
+
+        // TODO doesn't cover cases with multiple times eg minTimes and maxTimes - this is typically rare scenario
```

Comment:
Do we need to change this to `else if` for these statements? IIRC using sequential `if` statements as before would write each verification without issue.

Reply:
You know I had the same thought last night a few hours before your message and I messaged Tim to hold off on merge to add a commit. Will fix.